packages/wrangler/src/index.tsx

```diff
@@ -779,18 +750,16 @@ export async function main(argv: string[]): Promise<void> {
 
       // -- snip, extract --
 
-      if (!args.local) {
-        const loggedIn = await loginOrRefreshIfRequired();
-        if (!loggedIn) {
-          // didn't login, let's just quit
-          console.log("Did not login, quitting...");
-          return;
-        }
+      const loggedIn = await loginOrRefreshIfRequired();
```

Comment:
Is it possible to leave all the sections inside `// -- snip, extract --` and `// -- snip, end --` as is in this file? I'd like to replace them with an abstraction I'm working on, and had left them exactly the same so I could gather them at one go.

Reply:
All the blocks are identical now. The only change is that there is now not a check for `if (!args.local) {` inside the blocks.